test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -93,11 +106,10 @@ import Chainweb.Version
 import Chainweb.Version.Mainnet
 import Chainweb.Storage.Table.RocksDB
 
-
 -- -------------------------------------------------------------------------- --
 -- Global Settings
 
-nNodes :: Natural
+nNodes :: Word
```

Comment:
you'll see a few similar changes throughout this PR. there's no reason for this to be a `Natural`. we're never going to run greater than `maxBound :: Word` nodes on a single machine, or even come close to approaching it.